docs/source/data-types/udt.md

```diff
@@ -27,8 +35,9 @@ struct MyType {
 ```
 
 > ***Important***\
-> Fields in the Rust struct must be defined in the same order as they are in the database.
-> When sending and receiving values, the driver will (de)serialize fields one after another, without looking at field names.
+> Fields in the Rust struct must be defined in the same order and with the same names as they are in the database.
```

Comment:
I think this should be split into two remarks, one about deserialization (assumption about the order) and serialization (assumption that the names match). This sentence looks like a general recommendation while only parts of it apply to serialization, and others to deserialization.